Applications/CLI/Tests.cmake

```diff
@@ -527,6 +527,18 @@ if(NOT OGS_USE_MPI)
         expected_cube_1e3_pcs_0_ts_203_t_5.100000.vtu cube_1e3_pcs_0_ts_203_t_5.100000.vtu sigma_xx sigma_xx
     )
 
+    # Mechanics; Small deformations, Drucker-Prager (SDE-DP)
+    AddTest(
+        NAME Mechanics_SDE-DP_cube_1e0_displacementBC
```

Comment:
@wenqing thank you for the comment, the name has been changed as 'Mechanics_PlasticModel_Ehlers_SpecialCase_DruckerPrager'.